server/controllers/fast-food.js

```diff
@@ -1,21 +1,38 @@
-import FastFoodData from '../models/fast-food-data';
+import fastFoodData from '../models/fast-food-data';
 
 /**
  * Represents a Fast Food.
  */
 class FastFood {
-    /**
+  /**
      * This function gets all fastfood
      * @param {object} req - the request object.
      * @param {object} res - The response object.
      * @returns {object} Returns all orders information.
      */
-    getAllFoodOrders(req, res) {
-        if (!Object.keys(FastFoodData.foodOrders).length) {
-            return res.status(200).json({ orders: 'Food Orders is Empty' });
-        }
-        return res.status(200).json({ orders: FastFoodData.foodOrders });
+  getAllFoodOrders(req, res) {
+    if (!Object.keys(fastFoodData.foodOrders).length) {
+      return res.status(200).json({ orders: 'Food Orders is Empty' });
```

Comment:
I don't think it is advisable to return orders as a string here and then as an array below. I think it might be better to send back an empty array here, with/without a message.